internal/util/funcutil/parallel.go

```diff
@@ -98,6 +109,7 @@ func ProcessFuncParallel(total, maxParallel int, f func(idx int) error, fname st
 		select {
 		case err := <-errc:
 			close(quit)
+			wg.Wait()
```

Comment:
should call wg.Wait when count == routineNum

Reply:
OK